code/modules/mob/living/simple_animal/hostile/retaliate/crabapple.dm

```diff
@@ -0,0 +1,37 @@
+/mob/living/simple_animal/hostile/retaliate/crabapple
+	name = "crab apple"
+	desc = "No one likes crabs..."
+	icon_state = "crab_apple"
```

Comment:
I'd add some interaction with wirecutters/scissors, since regular crabs do have that behaviour.
Not asking that you do it, but didn't you make it a child of regular crabs?